packages/plugin-interaction-breadcrumbs/test/interaction-breadcrumbs.test.ts

```diff
@@ -1,7 +1,6 @@
 import plugin from '../src/interaction-breadcrumbs'
 
-import Client from '@bugsnag/core/client'
-import Breadcrumb from '@bugsnag/core/breadcrumb'
+import { Breadcrumb, Client } from '@bugsnag/core'
```

Comment:
Importing Client from `@bugsnag/core` also because otherwise in the test below the Breadcrumb prototype used in `c._breadcrumbs` is different to that references by `new Breadcrumb` because `import Client from '@bugsnag/core/client'` actually refers to the source file. Hence the above changes to the public types